agent/Bridge.js

```diff
@@ -396,15 +397,26 @@ class Bridge {
 
   _inspectResponse(id: string, path: Array<string>, callback: number) {
     var inspectable = this._inspectables.get(id);
-
     var result = {};
     var cleaned = [];
     var proto = null;
     var protoclean = [];
+
     if (inspectable) {
       var val = getIn(inspectable, path);
       var protod = false;
       var isFn = typeof val === 'function';
+
+      if (val && typeof val[Symbol.iterator] === 'function') {
+        var iterVal = Object.create({});  // flow throws "object literal incompatible with object type"
+        var count = 0;
+        for (const entry of val) {
```

Comment:
this can probably be done as a separate diff, I just would rather an extension update not make people's apps suddenly really slow under debugging